src/kernel/task.zig

```diff
@@ -92,6 +111,8 @@ pub const Task = struct {
             .stack_pointer = @ptrToInt(&k_stack[STACK_SIZE - 1]),
             .kernel = kernel,
             .vmm = task_vmm,
+            .file_handles = try bitmap.Bitmap(usize).init(VFS_HANDLES_PER_PROCESS, allocator),
```

Comment:
This can be a `ComptimeBitmap` as discussed above.

Reply:
Donezo :)